src/platform/Linux/BUILD.gn

```diff
@@ -27,6 +27,12 @@ if (chip_enable_openthread) {
   import("//build_overrides/ot_br_posix.gni")
 }
 
+if (chip_enable_nfc_based_commissioning) {
+  pkg_config("nfc_commissioning_config") {
+    packages = [ "libpcsclite" ]
```

Comment:
For other reviewers: `libpcsclite` is primarily licensed under BSD 3 clause license except for the unit tests and for an example code, which appears to be licensed under GPL v.3